torch/package/package_exporter.py

```diff
@@ -615,6 +615,11 @@ def close(self):
         if self.buffer:
             self.buffer.flush()
 
+        # If self.buffer is a BytesIO, rewind it in anticipation
+        # of a read in the near future.
+        if isinstance(self.buffer, io.BytesIO):
+            self.buffer.seek(0)
```

Comment:
I don't know if this is really that great of an idea, but it does help obviate the need to add `buffer.seek(0)` all over the place in unit tests.

Reply:
Probably not a good idea.
- Convention of Python standard library is not to do this (in particular `pickle` does not do this)
- We shouldn't assume ownership over the io stream.